pkg/email/template_service_impl.go

```diff
@@ -70,6 +70,9 @@ var modelPlanShareSubjectTemplate string
 //go:embed templates/model_plan_share_body.html
 var modelPlanShareBodyTemplate string
 
+//go:embed templates/shared_header.html
+var sharedHeaderTemplate string
+
```

Comment:
Guessing this will resolve itself when `main` is merged in, but seeing a lot of email template changes in this PR as-is

Reply:
Yes. I branched off of the shared email branch on accident 😮‍💨 Surprised there are diffs though, expected it to not have diff against main.